waltz-tools/src/main/java/com/wepay/waltz/tools/performance/PerformanceCli.java

```diff
@@ -165,6 +176,9 @@ protected void processCmd(CommandLine cmd) throws SubCommandFailedException {
                         throw new IllegalArgumentException("Found negative: lock-pool-size must be greater or equals to 0");
                     }
                 }
+                if (cmd.hasOption("up_to_date_watermark")) {
+                    partitionHighWaterMark = getHighWaterMark(partitionHighWaterMark, waltzClientConfig, numActivePartitions);
```

Comment:
NIT: method name should be `getHighWaterMarkPerPartition` to be more intuitive.

Reply:
Output of getHighWaterMark(..) method seems to be stored in partitionHighWaterMark variable. But why are we passing it as an argument? There is no other usage of this variable until this statement right which means it doesn't contain any information until then?